src/test/ui/transmute/transmute-from-fn-item-types-error.stderr

```diff
@@ -0,0 +1,75 @@
+error[E0591]: `unsafe fn() {bar}` is zero-sized and can't be transmuted to `isize`
+  --> $DIR/transmute-from-fn-item-types-error.rs:14:13
+   |
+14 |     let i = mem::transmute(bar);
+   |             ^^^^^^^^^^^^^^
+   |
+   = help: cast with `as` to a pointer instead
+
+error[E0591]: `unsafe fn() -> (isize, *const (), std::option::Option<fn()>) {foo}` is zero-sized and can't be transmuted to `*const ()`
```

Comment:
Should we change this error to also follow the same pattern of a simple header and notes to explain it?